.github/actions/setup-nvidia/action.yml

```diff
@@ -140,11 +148,107 @@ runs:
               )
           }
 
+          install_nvidia_driver_ubuntu20() {
```

Comment:
Looks like the function install_nvidia_driver_ubuntu20() is almost the same as install_nvidia_driver_amzn2() with some minor differences. Is it possible to use same common function or refactor this using smaller common functions ?

Reply:
Yes, will work on refactoring.